pipes/WDL/tasks/tasks_utils.wdl

```diff
@@ -721,8 +721,8 @@ task filter_sequences_by_length {
         Int    min_non_N = 1
 
         String docker = "quay.io/broadinstitute/viral-core:2.1.33"
+        Int    disk_size = 750
     }
-    Int disk_size = 300
```

Comment:
A thought for the future: I believe you could use the `size` function ([docs](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#float-sizefile-string)) in the future to make these a function of inputs, rather than requiring users to either rely on defaults or manually tweak them (both of which can be a little brittle when inputs change, as we found out!).

Reply:
Agreed -- yeah I think what's a little tricky is when it's not always easy to predict as a function of input size (ram especially) but disk is usually pretty easy to compute this way (2-3x the compressed input file size plus some constant baseline factor). This current PR does seem a little hastily evolved from just making it work, that said, I was surprised that much of anything changed downstream of the subsample step.

A few last tweaks coming in from @kraefrei soon I think.